ops/ccs-ops-misc/load_test/common/bfd_user_base.py

```diff
@@ -0,0 +1,155 @@
+'''Base class for Locust tests run against the FHIR endpoints.
+'''
+
+import json
+import random
+from typing import Callable, Dict, List
+
+from common import config, data, db, test_setup as setup, validation
+from locust import HttpUser
+import urllib3
+
+setup.set_locust_env(config.load())
+
+
+class BFDUserBase(HttpUser):
+    '''Base Class for Locust tests against BFD.
+
+    This class should automatically handle most of the common tasks that our
+    load tests require.
+    '''
+
+
+    # If a child class is going to use a set of data during the test, such as 
+    # Beneficiary IDs, MBI numbers, etc. then it has to be defined in this class
+    # field so that we can pre-load that data from the database.
+    DATA_REQUIRED = [
+        # 'BENE_IDS',
+        # 'MBIS',
+        # 'CONTRACT_IDS'
+    ]
```

Comment:
_super_ nit: Instead of strings here, maybe we could use an [enum](https://docs.python.org/3/library/enum.html)? I don't know if that would be more "pythonic", but it would potentially let us type hint this field (as something other than `str[]`) and allow for autocomplete and correctness checking. Also would eliminate the possibility of typoing one of the three data types and then wondering why the list of data is empty.

> Not sure if we do "nits" here, but in the past I've always prefixed comments on reviews that I'm fine with being ignored with "nit:".